### DISCUSSION TITLE
Aromatic Rings

### DISCUSSION THREAD

**Discussion by HadarAmit** [rdkit/rdkit#6362] [Q&A]
Hey,
I tried to find a substructure between molecules, and put the results in grid-image. In the output, there is a missing bond in the aromatic rings, but when I print just one substructure alone it prints this correct...

def defult():
l = 5
ligs = []
common_atoms_matrix = []
#make a list from the ligands that we insert to the code
for i in range(1, int(l)+1):
ligand = "ligand_" + str(i) + ".pdb"
ligs.append(ligand)
for i in range(len(ligs)):
mol_object = Chem.MolFromPDBFile(ligs[i])
Chem.SanitizeMol(mol_object)
ligs[i] = mol_object
return ligs

ligs = defult()
ls = []
for a, i in enumerate(ligs):
ks = []
for b, j in enumerate(ligs):
l = rdFMCS.FindMCS([i, j], bondCompare=rdFMCS.BondCompare.CompareAny).smartsString
l = Chem.MolFromSmarts(l)
#Chem.SanitizeMol(l)
ls.append(l)
Draw.MolsToGridImage(ls, molsPerRow=5, subImgSize=(250,250))

<img width="773" alt="צילום מסך 2023-05-11 132315" src="https://github.com/rdkit/rdkit/assets/133215803/4211b7ec-4020-41a6-8769-23c21505461d">

l = rdFMCS.FindMCS([Chem.MolFromPDBFile("ligand_5.pdb", sanitize = True), Chem.MolFromPDBFile("ligand_4.pdb", sanitize = True)], bondCompare=rdFMCS.BondCompare.CompareAny).smartsString
l = Chem.MolFromSmarts(l)
IPythonConsole.drawOptions.addBondIndices = True
l

<img width="398" alt="צילום מסך 2023-05-11 132559" src="https://github.com/rdkit/rdkit/assets/133215803/e417ce3d-d755-4f65-a549-fdef05056309">

why is this happening? and how can I solve this problem?
Thank you,
Amit

**Comment by greglandrum** [comment]
Hi @HadarAmit, I'm afraid I don't understand the question you're asking and the code you posted lost all formatting, so it's no longer legal python (if you enclose code in triple ` characters github will not format it). Could you please be more specific about what you are trying to do and what you think is going wrong?
